backend/open_webui/retrieval/web/brave.py

```
@@ -12,31 +12,41 @@
 def search_brave(
     api_key: str, query: str, count: int, filter_list: Optional[list[str]] = None
```

Ruff (0.12.2)

Unused function argument: `count`

(ARG001)